include/aspect/material_model/interface.h

```diff
@@ -1047,7 +1047,6 @@ namespace aspect
     }
 
 
-
```

Comment:
It is not really related to melt, but the line is removed on purpose (now there are two empty lines between each of the declarations).